rosetta/go.mod

```diff
@@ -0,0 +1,58 @@
+module github.com/axelarnetwork/axelar-core
```

Comment:
why do we need a separate go.mod?

Reply:
We use one repo to build 2 different images.
For Rosetta support we had a new prerequisite, we needed to use a Debian base image instead of alpine. see Dockerfile.rosetta.
That is the reason why we isolate all the "rosetta files" in a separate folder.

Reply:
Rosetta go.mod is not up-to-date? Can we use a Debian base image for the normal build too instead of Alpine?

Reply:
A couple of weeks ago, when we did the setup I think it was up to date. 
we can use a Debian base image if needed. There is a couple of modifications to do (use apt instead of apk, use Debian commands).